test/components/entity/activity.spec.js

```diff
@@ -64,11 +97,41 @@ describe('EntityActivity', () => {
         action: 'entity.create',
         details: sourceDetails
       });
-      const entries = mountComponent().findAllComponents(EntityFeedEntry);
+      const component = mountComponent();
+      component.findAll('.feed-entry-group').length.should.equal(1);
+      const entries = component.findAllComponents(EntityFeedEntry);
       entries.length.should.equal(2);
       entries[0].props().entry.action.should.equal('entity.create');
       entries[1].props().entry.action.should.equal('submission.create');
       entries[1].props().submission.instanceId.should.equal('some-uuid');
     });
   });
+
+  it('passes the correct entityVersion prop', () => {
```

Comment:
This is mostly a test of existing behavior that I thought would be useful to add. We're using `entityVersion` in more ways (now in `scrollData()`), so I wanted to verify that it's set correctly.